app/views/shared/_details.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= stylesheet_link_tag('style') %>
+<div class="container"> 
+ <div class="photo"> 
+  <img class="image" src=<%= user.photo %> alt="photo" /> 
+ </div> 
+ <div class="details"> 
+   <h3 class="name"><%= user.name %></h3> 
```

Comment:
- As this is not mentioned by the early reviewer, I am making this optional but I would highly recommend fixing the size of the pic as it doesn't look good in the UI. Please **fix it** before **merging**. As you can see how it looks with image.

![Screenshot from 2023-03-10 02-33-12](https://user-images.githubusercontent.com/26767383/224158484-1736b265-2988-48f7-b2b3-9e2fb1475aae.png)